config/tutorials-preview.json

```diff
@@ -0,0 +1,12 @@
+{
+	"extends": "base",
+	"io_sites": {
+		"max_static_paths": 1
+	},
+	"dev_dot": {
+		"max_static_paths": 1
+	},
+	"learn": {
+		"max_static_paths": 10000
```

Comment:
🤔 The paths are currently built from fathom analytics, it'd be worth validating that this does build all tutorial paths as expected since I'm not 100% sure what we get back from fathom.